lib/rex/post/meterpreter/extensions/stdapi/fs/file.rb

```diff
@@ -96,7 +98,8 @@ def File.search( root=nil, glob="*.*", recurse=true, timeout=-1 )
         files << {
           'path' => client.unicode_filter_encode(results.get_tlv_value(TLV_TYPE_FILE_PATH).chomp( self.separator )),
           'name' => client.unicode_filter_encode(results.get_tlv_value(TLV_TYPE_FILE_NAME)),
-          'size' => results.get_tlv_value(TLV_TYPE_FILE_SIZE)
+          'size' => results.get_tlv_value(TLV_TYPE_FILE_SIZE),
+	      'date' => results.get_tlv_value(TLV_TYPE_SEARCH_DATE)
```

Comment:
What date value is this? If it's an attribute of the file and not the search parameters then it should use be `TLV_TYPE_FILE_DATE`. Based on the table below it looks like this is the modified timestamp, so I'd suggest renaming it to `TLV_TYPE_FILE_MTIME`.

Reply:
It is the output mtime, so I'll rename it to `TLV_TYPE_FILE_MTIME` to make it clearer.